app/app.module.ts

```diff
@@ -10,20 +10,20 @@ import {NavbarComponent}  from './navbar/navbar.component';
 import {JumbotronComponent}  from './jumbotron/jumbotron.component';
 import {FooterComponent}  from './footer/footer.component';
 import {GoogleMapComponent}  from './google-map/google-map.component';
-import {WeatherComponent}  from './weather/weather.component';
-import {CityComponent}  from './city/city.component';
-import {SearchComponent}  from './search/search.component';
+import {WeatherComponent}  from './meteo/weather/weather.component';
```

Comment:
how about using ```index.ts``` files for exporting everything needed using one file?

it will look something like this
```
import {SearchComponent, WeatherComponent, CityComponent}  from './meteo/index';
```